doc/index.rst

```diff
@@ -56,3 +56,15 @@ Python API reference
 ====================
 
 .. automodapi:: astro_metadata_translator
+
+Command Line Utilities
```

Comment:
In the long run you might want to move this documentation out from the midst of the Python API references. Perhaps the solution is to drop the API Reference for `astro_metadata_translator.bin.translateheader` since it's unlikely that API wouldn't be needed. Or you might want to move the CLI reference pages to another rst file and link to with with a toctree? Just a thought.

Reply:
Given how little docs there are I think in the short term I will move this above the API section.